test-utils/shared/image.tsx

```diff
@@ -36,7 +36,7 @@ export const testSourceProps = <T,>({
         'mobile-source',
         'tablet-source-compressed',
         'tablet-source',
-        'display-source-compressed',
+        'desktop-source-compressed',
```

Comment:
what about `desktop-source`?

Reply:
Sorry, I missed the issue. It is no `desktop-source` in our component. Only `compressed` image in `source` tag on desktops